hexa/user_management/graphql/schema.graphql

```diff
@@ -14,6 +9,33 @@ type Avatar {
     initials: String!
     color: String!
 }
+type Team {
+    id: String!
+    name: String!
+    memberships(page: Int, perPage: Int): MembershipPage!
+}
+type TeamPage {
+    pageNumber: Int!
+    totalPages: Int!
+    totalItems: Int!
+    items: [Team!]!
+}
+enum MembershipRole {
+    ADMIN
+    REGULAR
+}
+type MembershipPage {
+    pageNumber: Int!
+    totalPages: Int!
+    totalItems: Int!
+    items: [Membership!]!
+}
+type Membership {
```

Comment:
I'm wondering if `TeamMembership` would not be better but it's just a thought.

Also I'd add `createdAt`, `updatedAt` on this relation, wdyt ?

Reply:
For `TeamMembership`, not now - the model is called `Membership` as well.

I've added the `createdAt` / `updatedAt` fields as suggested